.storybook/theme-decorators.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { withThemesProvider } from 'storybook-addon-styled-component-theme';
+import classic from '../src/style/themes/classic';
+import small from '../src/style/themes/small';
+import medium from '../src/style/themes/medium';
+import large from '../src/style/themes/large';
+import none from '../src/style/themes/none';
+
+/** @see https://stackoverflow.com/questions/925039/detect-iframe-embedding-in-javascript */
```

Comment:
I think that there is no need to link to an SO issue, and a simple comment will suffice, e.g.: `checking if the current window is embedded in an iFrame`

Reply:
Updated in 6fe8897 :thumbsup: